pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -283,14 +370,19 @@ func (ic *ifConfigurator) configureContainerLink(
 	mtu int,
 	sriovVFDeviceID string,
 	result *current.Result,
+	isSriovPodLink bool,
```

Comment:
This works for me too. I slightly prefer to use two arguments for bridge uplink VF ID and Pod link VF ID respectively, like: brSriovVFDeviceID and podSriovVFDeviceID, but would let you decide.

Reply:
Thanks for the suggestion. Just checking if the preference is to remove 'isSriovLink bool' and add 'podSriovVFDeviceID string' instead.

Reply:
Yes, I prefer to add `podSriovVFDeviceID  string`, and maybe change the existing `sriovVFDeviceID` to `brSriovVFDeviceID`

Reply:
Done